### PR TITLE
fix editor issues #452, #462, #463, #448 and some other minor fixes

### LINKED ISSUE #462: Display problem in vscode
With code 1.50.1, strange characters are written above and below "No goals".

### PR DESCRIPTION
**Pull request by firewall2142**
#462 -  Every new line character was encoded as the sequence `\n\xef\xbf\xbc` in mutli-line strings. 
The variables `header` and `footer` inside `buildGoalsContent(goals: Goal[])` function were initialized with multi-line strings. 
I am curious why this happened, even copy pasting the code in github comments had those strange characters. 

#463 - added alpha values

#448 - removed tslint from devDependencies (tslint depends on mkdirp, mkdirp depended upon minimist)

Other minor fixes:
* Moved css styles to seperate file
* Fixed the proof line

I think this was all for the editors for now.

### CODE REVIEW

**Comment by fblanqui** [comment]
Thank you @firewall2142 for this new PR!

**Comment by fblanqui** [comment]
With the Light color theme (white background), the goals are difficult to read. Could you please print hypothesis and goals in black in this case?

**Comment by fblanqui** [comment]
Thanks @firewall2142 to update your PR so that we can merge it.

**Comment by firewall2142** [comment]
@fblanqui I think this PR can be merged automatically without any issues

**Comment by firewall2142** [comment]
Oh I see. I had to update, my bad. I am not so experienced with github

**Comment by fblanqui** [comment]
Thank you @firewall2142 !

File: editors/vscode/src/client.ts [hunk 1]
```diff
@@ -29,23 +29,23 @@ export function activate(context: ExtensionContext) {
     //___Declaration of workspace variables___
 
     //Position of the proof cursor : colored highlights show until which point the proof was surveyed
-    let proofState : Position = new Position(1, 0);
+    let proofState : Position = new Position(0, 0);
```
Comment (ejgallego) [inline editors/vscode/src/client.ts:32]: Nice catch!

Comment (firewall2142) [reply editors/vscode/src/client.ts:32]: Thanks :-)

File: editors/vscode/src/client.ts [hunk 2]
```diff
@@ -487,6 +489,10 @@ function buildGoalsContent(goals : Goal[]) {
     footer =` </p>
 ￼       </body>
 ￼       </html>`;
+
+    // remove character sequences caused by multiline strings
+    header = header.replace(/[\u{0080}-\u{FFFF}]/gu,"");
+    footer = footer.replace(/[\u{0080}-\u{FFFF}]/gu,"");
```
Comment (ejgallego) [inline editors/vscode/src/client.ts:493]: Interesting; indeed we should improve the document type that LP is sending to the client. I have some ideas here which could provide for very cool display [see for example what jsCoq goal buffer can do already] Let's hopefully discuss on a call.

**Comment by ejgallego** [comment]
Nice work, thanks @firewall2142 !

**Pull request merged**